pkg/daemon/daemon.go

```diff
@@ -580,6 +580,14 @@ func (dn *Daemon) nodeHasAnnotation(annoKey string, value string) bool {
 
 func (dn *Daemon) isNodeDraining() bool {
 	if anno, ok := dn.node.Annotations[annoKey]; ok && (anno == annoDraining || anno == annoMcpPaused) {
+		// for openshift cluster draining should be true only if the annotation has MCP paused
+		if dn.openshiftContext.IsOpenshiftCluster() && !dn.openshiftContext.IsHypershift() {
+			if anno == annoMcpPaused {
```

Comment:
I dont know how MCP Pause annot is being used so i cannot comment on the correctess of this and how it would affect all palces which call this method


but some comment about the style :) 

you can check for `annoDraining` and drop the else.

Reply:
Hi @adrianchiris, I addressed your comment about style in:
- https://github.com/k8snetworkplumbingwg/sriov-network-operator/pull/421

I also added some unit tests to be sure we do not forget any use cases.
Can you please refer to that PR?